pkg/defaultmonitortests/types.go

```diff
@@ -107,8 +108,8 @@ func newDefaultMonitorTests(info monitortestframework.MonitorTestInitializationI
 
 	monitorTestRegistry.AddMonitorTestOrDie("image-registry-availability", "Image Registry", disruptionimageregistry.NewAvailabilityInvariant())
 
-	monitorTestRegistry.AddMonitorTestOrDie("apiserver-availability", "kube-apiserver", disruptionlegacyapiservers.NewAvailabilityInvariant())
-	monitorTestRegistry.AddMonitorTestOrDie("apiserver-new-disruption-invariant", "kube-apiserver", disruptionnewapiserver.NewDisruptionInvariant())
```

Comment:
these need to remain and be untouched.

Reply:
in order for us to compare the results side by side, we also need to run the new disruption tests against the external load balancer, this way we have apples to apples comparison. We want to answer the question how does the legacy and the new disruption test behave differently against the same load balancer.
So let's keep the legacy ones, and add the following tests using the new disruption library:
- a) kube-api-reused-http2-external
- b) kube-api-reused-http1-external
- c) kube-api-new-http2-external 
- d) kube-api-new-http1-external  

we already have legacy versions of `b` and `d` today, this will give us apples to apples comparison.